### PR TITLE
SET-708 Install RH CA on CCI VMs

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/SET-708

zeus-vars PR: https://gitlab.cee.redhat.com/jboss-set/zeus-vars/-/merge_requests/111

### CODE REVIEW

**Comment by spyrkob** [comment]
@RanabirChakraborty do you know if the molecule failure related to this change?

**Comment by RanabirChakraborty** [comment]
It doesn't look like that, it's happening because of the account role molecule test issue. Can you rerun the CI test again. If it fails I'll check it tomorrow.

File: cci_worker.yml [hunk 1]
```diff
@@ -64,3 +64,19 @@
         - old_releases
       loop_control:
         loop_var: taskfiles
+
+    - name: Ensure CA certificates directory exists
```
Comment (spyrkob) [inline cci_worker.yml:68]: @RanabirChakraborty that belongs to either `ssh` role or `cci_worker` role. Let's not put any more tasks in the playbooks

File: cci_worker.yml [hunk 2]
```diff
@@ -64,3 +64,19 @@
...
+
+    - name: Download RedHat CA certs
+      ansible.builtin.get_url:
+        url: "{{ cert.url }}"
+        dest: "{{ ca_certs_path}}/{{ cert.alias }}"
+      with_items: "{{ root_ca_certs }}"
+      loop_control:
+        loop_var: cert
+
+    - name: Update CA trust
```
Comment (spyrkob) [inline cci_worker.yml:81]: @RanabirChakraborty can you make this step either execute only if one of the files has changed?

Comment (RanabirChakraborty) [reply cci_worker.yml]: Thanks! @spyrkob updated the PR accordingly and the molecule fix PR is here - https://github.com/jboss-set/zeus/pull/241

**Review by spyrkob — APPROVED** [review]
LGMT, thanks @RanabirChakraborty

**Pull request merged**